app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,18 @@
+import React, { useEffect } from "react"
+import { useSelector, useDispatch } from 'react-redux';
+import { getGreetings } from "../configureStore";
+
+const Greeting = () => {
+  const state = useSelector((state) => state);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getGreetings());
+  }, []);
+  return(
+    <div>
+      <h1>{state.greeting}</h1>
+    </div>
+  )
+}
+
+export default Greeting
```

Comment:
- [ ] Kindly fix this issue when the page loads a greeting message should come from the redux store. however, currently, I am only seeing nothing on the screen. no action despatched, the store does not exist.
![image](https://user-images.githubusercontent.com/65068771/154177950-34a1b430-14e0-47d4-a496-40aa11f407e1.png)